### PR TITLE
protonvpn-cli: change to new upstream

### PR DESCRIPTION
**Pull request by HadetTheUndying**
ProtonVPN's upstream address changed, I have updated the template to make packaging it in the future less of a headache for the next release.

### CODE REVIEW

**Comment by Piraty** [comment]
a rebuild is not necessary i guess, so please don't revbump the package

File: srcpkgs/protonvpn-cli/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'protonvpn-cli'
 pkgname=protonvpn-cli
 version=2.2.2
-revision=1
+revision=2
```
Comment (Piraty) [inline srcpkgs/protonvpn-cli/template:4]: a rebuild is not necessary i guess, so please don't revbump the package

Comment (HadetTheUndying) [reply srcpkgs/protonvpn-cli/template:4]: I'll fix this first thing in the morning. I was unsure since the checksum changed.

Comment (HadetTheUndying) [reply srcpkgs/protonvpn-cli/template]: Should be good now. I'll keep that in mind for upstream changes in the future.

**Pull request merged**